.github/workflows/tests.yml

```diff
@@ -82,3 +85,10 @@ jobs:
         run: |
           # Requires async test dependencies
           pytest tests/adapter_tests/asgi/
+      - name: Install all dependencies
+        run: |
+          pip install -r requirements/testing.txt
```

Comment:
QQ: Should this be with the "Install all dependencies" job?

It's not super clear to me which steps use which dependencies and TBH I wouldn't be against separating `sync` from `async` tests if dependencies are shared in unexpected ways 👾